docs/consent/google-tag-manager.md

```diff
@@ -0,0 +1,35 @@
+# Google Tag Manager Consent Integration
+
+Support for Google Tag Manager is provided via GTM's data layer variable. When either the page loads or consent is subsequently changed the data layer is updated allowing you to create triggers based on the consent given.
```

Comment:
If we're talking about a variable, maybe we should name it specifically? e.g. "via GTM's `dataLayer` variable."